cligj/features.py

```diff
@@ -125,6 +125,8 @@ def iter_features(geojsonfile, func=None):
                     newfeat = func(to_feature(json.loads(line)))
                     if newfeat:
                         yield newfeat
+            else:
+                raise TypeError("Type must be Feature or FeatureCollection")
```

Comment:
Minor comment: the text in this error doesn't seem strictly true - it looks like `{"coordinates": ...}` will be accepted by line 120, which isn't a "Feature" or a "FeatureCollection".

Reply:
Updated to something more generic, "Invalid type" [dfd5fa3](https://github.com/mapbox/cligj/pull/40/commits/dfd5fa366d60d1f44ea923faae748472b4e4b6c4)